src/server/node_func.c

```diff
@@ -2388,9 +2399,12 @@ decode_Mom_list(struct attribute *patr, char *name, char *rescn, char *val)
 }
 
 
-#ifndef	WIN32
+static int	ndevices;
 static int	nsockets;
 
+#ifndef	WIN32
+static double	CrayVer;
```

Comment:
You are assuming the version may always be represented as a double. What about the case where major/minor/bugfix is specified?

Reply:
It should be basil_response_version. I'll change this. The supported versions are 1.4, 1.3, 1.2 and 1.1